### PR TITLE
Reorder and regroup module fields

### PR DESCRIPTION
**Pull request by kemmerle**
Since incorporating my and @levinkeo's changes to the module fields, I thought that the fields looked a bit disorganized while working in the page editor. 

I reordered and regrouped the different field groups for a (hopefully) more seamless editing experience. 

cc @achong07, @levinkeo (you might want to take a look to ensure that I've updated `moduleData` correctly in the `module.html` file).

### CODE REVIEW

File: src/modules/events-app.module/module.html [hunk 1]
```diff
@@ -16,8 +16,9 @@
     data-portal-id="{{ portal_id }}">
     {%- set moduleData = module -%}
     {%- do moduleData.update({
-      'events_root': content_by_id(moduleData.events_root).absoluteUrl,
-      'my_events_page': content_by_id(moduleData.my_events_page).absoluteUrl
+      'events_root': content_by_id(moduleData.page_roots.events_root).absoluteUrl,
+      'my_events_page': content_by_id(moduleData.page_roots.my_events_page).absoluteUrl,
+      'hero_img': moduleData.hero_image_group.hero_img
```
Comment (levinkeo) [inline src/modules/events-app.module/module.html:21]: Might be a nitpick, but I don't think we should update properties in the dict copied into `moduleData` unless we need to. Rather than aliasing `moduleData.hero_img` -> `hero.hero_image_group.hero_img`, I think we should just refer to `moduleData.hero_image_group.hero_img.src` in the JS.

Same goes for the `page_roots` fieldgroup. I think we should preserve the field group references when we call them in JS rather than copying them to flat properties. 

Unless we're deliberately trying to flatten moduleData (in which case I think we'd want to remove child dicts entirely), I think we should only `.update()` what needs to be modified.

In other words, I think moduleData is confusing like this:

```json
{
  "page_roots": {"events_root": null, "my_events_page": null},
  "events_root": null,
  "my_events_page": null,
  "hero_image_group": {
    "hero_img": { ... }
  },
  "hero_img": { ... }
}

```

I think instead we should just keep the fieldgroups and update them as needed:
```jinja2
  {%- do moduleData.page_roots.update({ ... }) -%}
```

Comment (levinkeo) [reply src/modules/events-app.module/module.html:21]: This ^ is coming from the thought that the JSON'd `moduleData` should have the same shape as the defined fields in `fields.json`, but if folks disagree, I'm open to different opinions. 

My primary opinion is I think we should pass one (flattened moduleData) or the other (same shape as the original dict) rather than a mix of the two.

Comment (kemmerle) [reply src/modules/events-app.module/module.html:21]: @levinkeo, I can definitely make these changes and keep the original structure of the module intact, no prob!
